src/app/CommandHandler.cpp

```diff
@@ -168,8 +168,8 @@ void CommandHandler::Close()
     // We must finish all async work before we can shut down a CommandHandler. The actual CommandHandler MUST finish their work
     // in reasonable time or there is a bug. The only case for releasing CommandHandler without CommandHandler::Handle releasing its
     // reference is the stack shutting down, in which case Close() is not called. So the below check should always pass.
-    VerifyOrDieWithMsg(mPendingWork == 0, DataManagement, "CommandHandler::Close() called with %zu unfinished async work items",
-                       mPendingWork);
+    VerifyOrDieWithMsg(mPendingWork == 0, DataManagement, "CommandHandler::Close() called with %u unfinished async work items",
+                       static_cast<unsigned int>(mPendingWork));
```

Comment:
mPendingWork should just be an `unsigned` instead of `size_t`.  It's a simple counter...  Followup for making that change is fine.  If that happens, the other logs that involve it in this file should be fixed too.

Reply:
Got it, thanks